src/foam/core/SubX.java

```diff
@@ -0,0 +1,85 @@
+package foam.core;
```

Comment:
Copyright

Reply:
Added in https://github.com/foam-framework/foam2/pull/5037/commits/f70dab822dfbab457d3f23fe0c879fbd6619f160.